### PR TITLE
Cleanup Dockerfile, remove uid hack, remove dependency by copying sou…

### PR DESCRIPTION
**Pull request by delamonpansie**
…rces into docker

### CODE REVIEW

File: docker/docker-entrypoint.sh [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
-    chown ${UID} /home/jenkins/.bash* || true
-fi
+cd arakoon
 
 # finally execute the command the user requested
-command="cd arakoon && ./docker/suites.sh $@"
-sudo -E -i -u jenkins bash -l -c "$command"
+case $1 in
+  bash)
+	bash
```
Comment (domsj) [inline docker/docker-entrypoint.sh]: if we add "$@" here we can run arbitrary commands in the container?

**Review by domsj — APPROVED** [review]

File: docker/docker-entrypoint.sh (hunk 1 above)
Comment (delamonpansie) [reply docker/docker-entrypoint.sh]: yes, we can.
the syntax would be like `./docker/run.sh ubuntu-16.04 bash -c top`
I'd rather change bash to sh, actually.

**Comment by domsj** [comment]
https://github.com/openvstorage/arakoon/blob/1.9/COMPILING#L23
this line of documentation will no longer be correct after the PR is merged?

**Comment by delamonpansie** [comment]
Good catch, thanks.

**Review by domsj — APPROVED** [review]

**Pull request closed without merging**